.github/workflows/CI.yml

```diff
@@ -3,22 +3,29 @@ name: CI
 on:
   push:
     paths-ignore:
-      - '.travis.yml'
-      - 'filecopy.bat'
+      - '.github/*'
+      - '.github/*_TEMPLATE/**'
+      - '*.bat'
+      - '*.yml'
+      - '*.md'
       - 'LICENSE*'
-      - 'README.md'
   pull_request:
     paths-ignore:
-      - '.travis.yml'
-      - 'filecopy.bat'
+      - '.github/*'
+      - '.github/*_TEMPLATE/**'
+      - '*.bat'
+      - '*.yml'
+      - '*.md'
       - 'LICENSE*'
-      - 'README.md'
+  workflow_dispatch:
```

Comment:
What's this for?

Reply:
https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/